### DISCUSSION TITLE
dynamic filter for relation column is work only for first page

### DISCUSSION THREAD

**Discussion by fadliasyhariStar** [Budibase/budibase#12615] [Help] [answered]
Let's say i have table person and management. Person to management is one to many relationship. Then I want to present a management table with limit 10 that also included person.name. When i try to use dynamic filter to filter the person.name it's only filtering the first page. Is there a way to fix this? 
Thanks.

![Screencast from 19-12-23 11_20_54](https://github.com/Budibase/budibase/assets/134992316/d2dd194b-05b0-407b-b1b1-376c3fc81488)

I'm using version 2.13.9. Selfhosted. Postgres datasource.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @fadliasyhariStar,

One of our engineers gave an explanation to an issue similar to what you've showcased [here](https://github.com/Budibase/budibase/issues/10843#issuecomment-1645640573).

Looking at this again this could be different. Could you create an bug for this, including as much information as possible e.g. the table create schemas, an app export etc. So we can easily test this, I'm hoping that it won't be a similar answer to the link provided above as its not searching based on a relationship but rather just text.

Thanks in advance for doing this.
